components/multimedia/dav1d/Makefile

```diff
@@ -0,0 +1,57 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2021 (c) Tim Mooney. All rights reserved.
+#
+BUILD_BITS=		32_and_64
+BUILD_STYLE=		meson
```

Comment:
Change BUILD_BITS to
BUILD_BITS= 64_and_32
This will bring the 64-bit program(s( into /usr/bin and 32-bit ones into /usr/$(MACH32)/bin.
Then you can filter out the 32-binaries in the manifest file.
Background: We only want 64-bit binaries and try to get rid of 32-bit binaries wherever possible.
Maybe it's even simpler here as we don't have any existing consumers and we can also change ffmpeg to make it 64-bit only. We provide 32-bit libraries where necessary for backwards compatibility.

Reply:
I'm not opposed to switching ffmpeg to 64 bit only, and two of its dependencies (minidlna and vlc) appear to already be 64 bit only, but gst-libav1 would also need to switch to be 64 bit only.  I'm also not opposed to that, but it does leave us with the weird situation where most of gstreamer1 is 32 bit and 64 bit, but the libav stuff would be 64 bit only.

Reply:
Partially fixed.  Switched it to `64_and_32`, rebuilt the `sample-manifest` and updated the `dav1d.p5m` file, skipping the one 32 bit binary.

Once you've weighed in on whether it's OK to have `gst-libav1` switch to be 64 bit only and be unlike the other `gstreamer` components then I'll know if any additional changes are needed or the current commit addresses all concerns.